agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -247,6 +258,10 @@ private TelemetryItem exportRemoteDependency(SpanData span, boolean inProc, long
       applySemanticConventions(telemetryBuilder, span);
     }
 
+    if (isPreAggregated) {
```

Comment:
instead of passing around `isPreAggregated`, does it make sense to call `isPreAggregated(span)` here?